File: Prafful-Bisht/DAY-2/Day-2-B.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        int i=0,j=0,k=0;
        vector<int>nums(m+n);
        while(i<m && j<n){
            if(nums1[i]>=nums2[j]){
            nums[k]=nums2[j];
            k++;j++;
            }
            else{
                nums[k] = nums1[i];
                k++;i++;
            }
        }
        while(i<m){
            nums[k]=nums1[i];
            k++;i++;
        }
        while(j<n){
            nums[k]=nums2[j];
            k++;j++;
        }
        for(int p=0;p<m+n;p++){
            nums1[p]=nums[p];
        }
    }
};

File: Mukul Bhalla/Day 2/Day 2 B.py
class Solution(object):
    def merge(self, nums1, m, nums2, n):
        """
        :type nums1: List[int]
        :type m: int
        :type nums2: List[int]
        :type n: int
        :rtype: None Do not return anything, modify nums1 in-place instead.
        """
        count = 0
        for i in range(0, m+n):
            if (i > m-1):
                nums1[i] = nums2[count]
                count += 1
        nums1.sort()
